index.bs

```diff
@@ -291,7 +297,14 @@ A <dfn>font representation</dfn> is some concrete representation of a font. Exam
 </div>
 
 <aside class=note>
-The [=/font representation=]'s [=font representation/data bytes=] are generally expected to be the exact byte-by-byte representation of font files on the user's filesystem. UAs aren't expected to normalize the font data, so font representations would not vary across user agents for a given user on a particular OS. The lack of normalization supports the goal of enabling web applications that perform text rendering for content creation with the full fidelity of the font.
+As an example, the [=/font representation=] of an OpenType [[!OPENTYPE]] font would be as follows:
```

Comment:
Maybe:
```suggestion
This specification doesn't precisely define the values of the above fields for any particular font format, so that differences between operating systems don't make UAs non-compliant. However, for an OpenType [[!OPENTYPE]] font the following values would be sensible:
```